project/src/components/private-route/private-route.tsx

```diff
@@ -0,0 +1,19 @@
+import {Navigate} from 'react-router-dom';
+import {AppRoute, AuthorizationStatus} from '../../constants';
+
+type PrivateRouteProps = {
+  authorizationStatus: AuthorizationStatus;
+  children: JSX.Element;
+}
+
+function PrivateRoute(props: PrivateRouteProps): JSX.Element {
```

Comment:
Используй деструктуризацию сразу
```suggestion
function PrivateRoute({authorizationStatus, children}: PrivateRouteProps): JSX.Element {
```